Guidelines.md

```diff
@@ -1877,16 +1880,16 @@ note right of App Server: Update status and cache new "since" token
 
 === End Text ===
 ```
-[IANA-headers]: http://www.iana.org/assignments/message-headers/message-headers.xhtml
-[rfc7231-7-1-1-1]: https://tools.ietf.org/html/rfc7231#section-7.1.1.1
-[rfc-7230-3-1-1]: https://tools.ietf.org/html/rfc7230#section-3.1.1
-[rfc-7231]: https://tools.ietf.org/html/rfc7231
+[fielding]: https://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm
+[rfc-2616]: http://www.ietf.org/rfc/rfc2616.txt
```

Comment:
:exclamation: I believe this was correct before (it reverts a change made in #41) (applies to several links listed below this line as well)